test/JDBC/expected/BABEL-2875.out

```diff
@@ -7,8 +7,8 @@ CREATE TABLE t_sksql(
 )
 go
 
-SELECT SUBSTRING( T.name, 1, 20 ) AS "UDDT name",
-SUBSTRING( T2.name, 1, 20 ) AS "system dt name",
+SELECT pg_catalog.SUBSTRING( T.name, 1, 20 ) AS "UDDT name",
+pg_catalog.SUBSTRING( T2.name, 1, 20 ) AS "system dt name",
```

Comment:
why do we need to make these changes?

Reply:
Don't need these changes anymore, reverted them